.github/workflows/prepare-release.yml

```diff
@@ -13,3 +13,42 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: false
+
+      - name: Update Changelog
+        id: update-changelog
+        uses: realm/ci-actions/update-changelog@dac063f8ebd86c201590d07536d6f56f86f66ba9
+        with:
+          changelog: ${{ github.workspace }}/CHANGELOG.md
+
+      - name: Update pubspec.yaml
+        uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d #! 2.0.0
+        with:
+          find: 'version: .*'
+          replace: 'version: ${{ steps.update-changelog.outputs.new-version }}+alpha'
+          include: '**/pubspec.yaml'
+
+      - name: Update realm.podspec
+        uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d #! 2.0.0
+        with:
+          find: "  s.version(\\s+)= '[^']*'"
+          replace: "  s.version$1= '${{ steps.update-changelog.outputs.new-version }}+alpha'"
+          include: '**/realm.podspec'
+
+      - name: Update realm_core.libraryVersion
+        uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d #! 2.0.0
+        with:
+          find: "  String get libraryVersion => '[^']*';"
```

Comment:
Should we fail CI if we can not find this line. Just to be sure we don't refactor this code

Reply:
Added a check for that